### ISSUE TITLE
Question: handling JSON payload from server

### ISSUE THREAD

**Issue by ohenley** [WebReflection/hyperhtml-app#5]
Let say a request to mydomain.com/json-data returns some Json object (same form as array_of_stuff)
The following code works but I was wondering if there is a more 'idiomatic' way to do.

Mainly, is router.use(...) + $.getJSON the way to go? 

I tried to use router.get but I could not figure how to access the response data. My solution ends up specifying the path /json-data three times: router.navigate('/json-data')}, router.use('/json-data and $.getJSON( "/json-data". I suspect I am missing something here...

Thx!
 
```js
const router = new hyperHTML.app();
const appEl = document.getElementById('app');
var array_of_stuff = [
            { 
              Label: 'one', 
              Value: 1 
            }, 
            { 
              Label: 'two',
              Value: 2 
            }, 
            { 
              Label: 'three', 
              Value: 3 
            }
          ];

router.use('/json-data', () => {
  $.getJSON( "/json-data", function( data ) {
    render(data)
  });
});

function render(arr) {
  hyperHTML.bind(appEl)`
    <table class="table table-striped table-bordered table-sm">
    <thead>
      <tr><th>Label</th><th>Value</th></tr>
    </thead>
    <tbody>
      ${arr.map(obj => hyperHTML.wire(obj)`<tr><td>${obj.Label}</td><td>${obj.Value}</td></tr>`)}
    </tbody>
  </table>
  <button onclick=${() => router.navigate('/json-data')}>
      See json-data
  </button>`
}

render(array_of_stuff)
```

**Comment by WebReflection** [comment]
`router.get` is to define get requests so I'm not sure why one would use it to read anything.

TL;DR sorry for the late reply but yes, that's the way to go
